### PR TITLE
feat: Data consumers prefer text labels for download and viewer (#3465)

### PR DESCRIPTION
**Pull request by frano-m**
- #3465 

### Reviewers
**Functional:** 
@tihuan 

---

## Changes
- Replaced the download and explore svg icon buttons with text buttons "Download" and "Explore" as per [mock](https://www.figma.com/file/5X1Y2CHUjLOIu6MiuMsut1/Consortia-Collection?node-id=11%3A31357).
- See slack discussions [here](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1666800222817319?thread_ts=1666212743.525989&cid=C02CF7MQPGV) and [here](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1666737115461729?thread_ts=1666212743.525989&cid=C02CF7MQPGV) for the following adjustments to requirements:
  - The text "Viewer" should be "Explore".
  - Remove the unwanted tooltip e.g. "Download" and "Explore". Note, the tooltip "Exploration is currently unavailable for datasets with more than 2 million cells" remains.
  - The update to the button styles (primary and primary outlined) will be reflected in other buttons such as "Publish" and "Add" on the collection page and "Save" in the create collection modal.
  - Table cell line height has been increased to 20px.
  - The "Delete Collection" button styles on the collection page has been updated to match the adjacent buttons.

### CODE REVIEW

File: frontend/src/components/Collection/components/CollectionDescription/index.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export default function CollectionDescription({
         {description}
       </DescriptionText>
       {isModeActivated(ellipsisMode) && (
-        <StyledPrimaryAnchorButton
+        <StyledPrimaryMinimalButton
```
Comment (frano-m) [inline frontend/src/components/Collection/components/CollectionDescription/index.tsx:60]: Refactored name of button - technically this is not an anchor button!

File: frontend/src/views/Collection/components/ActionButtons/components/DeleteButton/index.tsx [hunk 2]
```diff
@@ -29,7 +31,7 @@ const DeleteCollectionButton = ({
 
   return (
     <>
-      <Button
+      <StyledOutlineButton
```
Comment (frano-m) [inline frontend/src/views/Collection/components/ActionButtons/components/DeleteButton/index.tsx:34]: Modified the style for "Delete Collection" button to match the adjacent buttons styles.

File: frontend/src/components/Collection/components/CollectionDatasetsGrid/components/Row/DatsetRow/index.tsx [hunk 3]
```diff
@@ -232,21 +231,18 @@ const DatasetRow: FC<Props> = ({
           {hasCXGFile(dataset) && (
             <Tooltip
```
Comment (frano-m) [inline frontend/src/components/Collection/components/CollectionDatasetsGrid/components/Row/DatsetRow/index.tsx:232]: Disable tooltip unless the max cell count has been exceeded.

**Comment by tihuan** [comment]
Hey @frano-m thanks so much for the PR! I see that this is in draft, so probably not ready for review yet? Thank you 😄

**Comment by frano-m** [comment]
Hey @tihuan, sorry I've only seen your message now... I was waiting for some confirmation on button positioning... It's good to go now thank you!

File: frontend/src/components/common/Button/common/style.ts [hunk 4]
```diff
@@ -1,65 +1,90 @@
-import { Button, Classes } from "@blueprintjs/core";
```
Comment (tihuan) [inline frontend/src/components/common/Button/common/style.ts:1]: wow awesome refactoring 🤩 ✨ Thanks so much for doing this!

File: frontend/src/components/common/Grid/components/ActionsCell/style.ts [hunk 5]
```diff
@@ -1,9 +1,8 @@
 import styled from "@emotion/styled";
 
 export const Actions = styled.div`
+  align-items: center; /* required for the "more" dropdown button to center align with buttons i.e. "Download" and "Explore" */
```
Comment (tihuan) [inline frontend/src/components/common/Grid/components/ActionsCell/style.ts:4]: thanks for the comment!

**Review by tihuan — APPROVED** [review]
Thanks so much for the awesome consolidation and refactoring, @frano-m 🚀 👏 Really appreciate it! 🙏

**Pull request merged**